client/web/src/person/PersonLink.tsx

```diff
@@ -50,6 +51,7 @@ export const PersonLink: React.FunctionComponent<
                 : person.email
         }
     >
+        <VisuallyHidden>Github User:</VisuallyHidden>
```

Comment:
```suggestion
        <VisuallyHidden>User:</VisuallyHidden>
```

Not always going to be GitHub